### ISSUE TITLE
Release description in detail.

### ISSUE THREAD

**Issue by apoorvmote** [aws-amplify/amplify-cli#3736]
**Which Category is your question related to?**

Amplify cli

**Amplify CLI Version** 

4.17.0

**What AWS Services are you utilizing?**

Amplify Congnito Dynamodb

**Provide additional details e.g. code snippets**

Just few hours ago there was release update. But I am interested in specifically what is changed in this release. You guys update documentation directly. I have to read every single page of documentation to figure out what is changed and what is new in this release. 

It would be nice if there was summery of changes with links to documentation to read in detail. 

Just to give you an example. I wouldn't know that you added support for dynamodb streams unless I read through all documentation. There is no section just for added new features.

**Comment by attilah** [comment]
@apoorvmote thanks for the feedback, I am marking it as an enhancement, we're working on to making it better. We are using conventional commits, so we've a per package ```CHANGELOG.md``` file where the changes can be gathered.

**Comment by apoorvmote** [comment]
@attilah `CHANGELOG.md` is hard to find and difficult to understand cryptic statements. If you create blog post or news post & just write summery in plain english with links to documentation to read more information then it would be great.

**Comment by renebrandel** [comment]
Hi - we're now leveraging Github releases. Review the latest release to view its changelog. The changelog includes changes from every downstream package.
